site/public/js/ta-grading-v2.js

```diff
@@ -93,16 +92,29 @@ $(function () {
       return;
     }
     const isPanelOpen = $('#' + panelId).is(':visible') && btnCont.hasClass('active');
-    // If panel is not in-view and two-panel-mode is enabled show the drop-down to select position,
+    // If panel is not in-view and two/three-panel-mode is enabled show the drop-down to select position,
     // otherwise just toggle it
-    if (isPanelOpen || !(taLayoutDet.isTwoPanelsEnabled && !isMobileView)) {
+    if (isPanelOpen || +taLayoutDet.numOfPanelsEnabled === 1) {
       setPanelsVisibilities(panelId);
     } else {
       // removing previously selected option
       selectEle.val(0);
       selectEle.is(':visible') ? selectEle.hide() : selectEle.show();
     }
   });
+
+  // panel position selector change event
+  $(".grade-panel .panel-position-cont").change(function() {
+    let panelSpanId = $(this).parent().attr('id');
+    let position = $(this).val();
+    console.log(position);
+    if (panelSpanId) {
+      const panelId = panelSpanId.split('_btn')[0];
+      setPanelsVisibilities(panelId, null, position);
+      $('select#' + panelId + '_select').hide();
```

Comment:
Submitty CSS Id's should use dashes instead of underscores
https://submitty.org/developer/coding_style_guide/css

However, when I was implementing the notebook code I started using dashes before I realized how much of the logic depended on the specific naming scheme. If it's too difficult to remove for now it can be refactored/made a future issue later.

I began modifying some of the logic to use `-` instead on https://github.com/Submitty/Submitty/pull/5681 so there will be some overlap and merge conflicts most likely

Reply:
Its actually depended on `_` for the DOM manipulation and is also used in the tests the same way... also I'm working on some other issue for TA-grading page, so, I will add refactor(`-` over `_`) these in that PR.
Let's not worry about the merge conflicts I will update the logics which you are mentioning, thanks:)